CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Moves job controller to the same Kubernetes pod with the
+  REANA-Workflow-Engine-* (sidecar pattern).
+- Adds support for HTCondor and Slurm compute backends.
```

Comment:
Move Condor and Slurm to the first place, this will be release highligth.  You can also add "Adds generic job manager class and provides example classes for CERN HTCondor and CERN Slurm clusters" to make it explicit that this is  for CERN only for now.